packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/DiagramPanel.tsx

```diff
@@ -89,6 +112,9 @@ export const DiagramPanel = ({
           <IconButton size="small" onClick={handleShare}>
             <ShareIcon />
           </IconButton>
+          <IconButton size="small" onClick={handleExport}>
+            <ImageIcon />
+          </IconButton>
```

Comment:
Can you add a tooltip like the reveal hidden and reveal faded actions?

Reply:
Yes I can